articles/quickstart/webapp/_includes/_getting_started.md

```diff
@@ -2,10 +2,10 @@
 **New to Auth?** Learn [How Auth0 works](/overview), how it [integrates with Regular Web Applications](/architecture-scenarios/application/web-app-sso) and which [protocol](/flows/concepts/auth-code) it uses.
 :::
 
-<%= include('../../../_includes/_new_app') %>
+<%= include('../../../_includes/_new_app', { showClientSecret: true }) %>
```

Comment:
This will add the Client Secret bullet point on all webapps. I double-checked each one to make sure it's used.